packages/terra-graphs/src/Graphs.module.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
Should we setup the theming structure as well?

Reply:
Wouldn't hurt to setup an example. I probably should have done that with our generator when i was updating it.

Reply:
I've done this: https://github.com/cerner/terra-graphs/pull/1/commits/e9d6aec8c6afbb208a75de90779ea8e7aea00345